src/data/solution-context.js

```diff
@@ -135,4 +135,15 @@ export default class SolutionContext extends BaseContext {
             console.error(error);
         }
     }
+    async userImpact(solution, user) {
+        try {
+            console.log('wat te doen met user? ' + user);
```

Comment:
Dit kan beter @jeetjeweet 
```suggestion
            console.log('wat te doen met user? ' + user);
```

Reply:
Gefixt a mattie @Tjerk132